core/domain/config_domain.py

```diff
@@ -257,3 +277,11 @@ def update_moderator_ids(committer_id, moderator_usernames):
 PROMO_BAR_MESSAGE = ConfigProperty(
     'promo_bar_message', UNICODE_SCHEMA,
     'The message to show to all users if the promo bar is enabled', '')
+
+VMID_SHARED_SECRET_KEY_MAPPING = ConfigProperty(
+    'vmid_shared_secret_key_mapping', VMID_SHARED_SECRET_KEY_SCHEMA,
+    'VMID and shared secret key corresponding to that VM',
+    [{
```

Comment:
Default values are used in local deployment. So when someone deploys Oppia-ml locally they dont have to go to Oppia and add VMID and shared key. This way we can reduce complexity of installing and running Oppia-ml in local development. Otherwise every time you start Oppia you have to go and add keys so that Oppia-ml can work with it.
Does that clarify why I added default value?